autoswitch_virtualenv.plugin.zsh

```diff
@@ -197,10 +197,13 @@ function check_venv()
                 if type "poetry" > /dev/null && _activate_poetry; then
                     return
                 fi
-            else
+            elif [[ -f "$venv_path" ]]; then
                 local switch_to="$(<"$venv_path")"
                 _maybeworkon "$(_virtual_env_dir "$switch_to")" "virtualenv"
                 return
+            elif [[ -d "$venv_path" ]] && [[ -f "$venv_path/bin/activate" ]]; then
```

Comment:
could you add some code comments explaining what is going on in each of these branches? (Just like you explained in the PR description)

Reply:
Sure, done.